client_plugin/utils/refcount/refcnt.go

```diff
@@ -387,7 +380,7 @@ func (r *RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 		log.Debugf("  Mounts for %v", ct.Names)
 		for _, mount := range containerJSONInfo.Mounts {
 			// check if the mount location belongs to vmdk plugin
-			if isVMDKMount(mount.Source) != true {
+			if isVMDKMount(mount.Driver) != true {
```

Comment:
> I mean if they install the vsphere plugin and call it by another org's name that won't be right.

They can have a good enough reason to name the alias something else. May be as part of automated deployment itself. The way I look at alias is basically to provide the convenience to user to give a name of choice. Limiting it to vsphere is too tight.
Also we have different name for shared plugin. @luomiao 

Second issue I see is with the existing customers who have  aliased the plugin something else. As part of plugin upgrade, they ought to preserve the alias and they can't switch to vsphere. They'll have to uninstall and install it.

Given the limited alternatives we have, it boils to the question as:
Probability of a customer not using "/var/lib/docker " for runtime (this is by default) VS probability of customer not naming the alias to vsphere.
We can afford the one with lesser probability.

I have seen neither of such cases so I don't have a choice.